src/Consolonia.Core/Drawing/RenderTarget.cs

```diff
@@ -69,7 +69,7 @@ void IDrawingContextLayerImpl.Blit(IDrawingContextImpl context)
 
         bool IDrawingContextLayerImpl.CanBlit => true;
 
-        public IDrawingContextImpl CreateDrawingContext()
+        public IDrawingContextImpl CreateDrawingContext(bool useScaledDrawing)
```

Comment:
Shouldn't we throw NotImplemented when argument is true? Just to notify we do not support scaledDrawing.